### ISSUE TITLE
Detects Plex and My Content, but No Rich Presence?

### ISSUE THREAD

**Issue by MAXimized490** [Phineas05/discord-rich-presence-plex#9]
Hi!

I've set up the application and all seems fine, but Rich Presence does not show up on Discord?
I even tried using pyInstaller to make it an executable since I know that things like EasyRP need to be selected as a game in order to work, but still no luck. Any ideas? Am I doing something wrong?
I'm running Windows 10 with Python 3.6.4 64-bit

Console Screenshot:
![image](https://user-images.githubusercontent.com/36754183/44624412-9491de00-a8b3-11e8-9323-dce98e339c59.png)

Folder:
![image](https://user-images.githubusercontent.com/36754183/44624414-c2772280-a8b3-11e8-8b4e-726407d41277.png)

Discord:
![image](https://user-images.githubusercontent.com/36754183/44624418-d91d7980-a8b3-11e8-8d60-7d2fcd8465a4.png)

Thanks in advance!

**Comment by Phineas05** [comment]
It won't work if you make it an executable and add it as a game on Discord. The script creates a separate process to pass to Discord and the executable added as a game might be overriding the rich presence of this process.

According to your console screenshot, the plexapi module doesn't seem to be receiving alerts about your Plex activity. Another person recently had the same issue (#8) and we found out that it was caused by the latest version of websocket-client which is used by plexapi. Install websocket-client version 0.48.0 by running `pip install websocket-client==0.48.0` and let me know if it works.

**Comment by MAXimized490** [comment]
Ah, okay! One step closer! I installed WebSocket Client and now I'm seeing the console say it's found a session and whatnot. Unfortunately, there seems to be a permission error? Something with HandShake.
![image](https://user-images.githubusercontent.com/36754183/44635490-77204b00-a96b-11e8-9044-40c797b983e7.png)

**Comment by Phineas05** [comment]
Try running it as an administrator.

**Comment by wdaniel526** [comment]
I am having the same issue, nothing seems to fix it. I even downgraded my version of Python

**Comment by wdaniel526** [comment]
I am having the same issue, nothing seems to fix it. I even downgraded my version of Python.
This has been happening ever since I updated the script to this newer version

![python_2018-09-10_15-50-37](https://user-images.githubusercontent.com/1734051/45328565-59c5b200-b511-11e8-8ee3-1d5e235cb606.png)

**Comment by Phineas05** [comment]
Did you downgrade websocket-client to 0.48.0 by running `pip install websocket-client==0.48.0`? The script isn't receiving alerts from Plex and that happens when you're using the newest version of websocket-client. Run `pip show websocket-client` to check which version you're using.

**Comment by wdaniel526** [comment]
Thank you, this fixed the issue

**Comment by MAXimized490** [comment]
Sorry for dropping this thread. I have gotten your script to work and all is well except for the fact that Discord does not recognize it when adding it as a game. No errors in the console regarding this, so I'm not sure what's going on.
![image](https://user-images.githubusercontent.com/36754183/50262972-34cac000-03d9-11e9-8140-00f86a5143a9.png)

**Comment by Phineas05** [comment]
That's normal, you're not required to add it as a game manually.

**Comment by MAXimized490** [comment]
Oh, my bad. Well, I decided to re-download since my copy of the script is a few months old, but still nothing. It runs fine and detects everything, but I'm getting nothing on Discord. Any ideas?

**Comment by Phineas05** [comment]
Can you send a screenshot of the console?

**Comment by MAXimized490** [comment]
Yes, but I've just made an interesting development. Previously, I was using my email as the username to connent to my account, since I had the setting to use my username on login disabled. Apparently, this must have been interfering somehow. I am now using my username to login and the console is completely different. Seems like everything worked except for a handshake with Discord?
![image](https://user-images.githubusercontent.com/36754183/50370456-8e172880-056c-11e9-9436-8207738f50b7.png)

**Comment by Phineas05** [comment]
Method of logging in doesn't affect it. Not sure why this would be required but can you try running it as an administrator? Run command prompt as an administrator and run the script through it by running `python <script file path>`.

**Comment by Phineas05** [comment]
That's strange since it works for me and others without administrator privileges. Not sure what could be wrong but good enough for now as long as it works this way I guess.
